app/models/item.rb

```diff
@@ -0,0 +1,10 @@
+# typed: strict
+class Item < ApplicationRecord
+  belongs_to :collection
+  has_many :customizations
+
+  validates :name, presence: true
+  validates :price, numericality: { greater_than_or_equal_to: 0 }
+  validates :discounted_price, numericality: { greater_than_or_equal_to: 0 }
+  validates :collection_id, presence: true
+end
```

Comment:
The Item model should validate that the 'discounted_price' is less than or equal to the 'price' to maintain data integrity. This can be done with a custom validation method.